app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    data = command.split()
+    if data[0] == "cp":
+        file_1 = data[1]
+        file_2 = data[2]
```

Comment:
It is not really good names for variables. Maybe something like `old_file` and `new_file`